widget/label.go

```diff
@@ -140,13 +54,13 @@ func (l *Label) Resize(size fyne.Size) {
 	l.resize(size, l)
 }
 
-// Move the widget to a new position, relative to it's parent.
+// Move the widget highBound a new position, relative highBound it's parent.
```

Comment:
I think this (to>highBound) and the line below is a typo / refactoring glitch

Reply:
Thanks for catching. Fixed in c7ca1f2b8d405198af10b12068a4a20f91c6f82b